classes/that.sc

```diff
@@ -140,15 +144,15 @@ That {
 		}, oscChannelName).fix;
 	}
 
-	*prCreateTrigger {|in, defaultTrigger, triggerFunction|
+	*prCreateTrigger { |in, defaultTrigger, triggerFunction|
```

Comment:
did you do this manually or is there some linter for this? I used the SC IDE internal formatter and it did not fix this behavior.

Reply:
I do it manually, there is no linter because there is no syntax universally agreed on. So a matter of taste …

Reply:
It would be maybe a good thing to enforce global styleguides via a linter as it avoids discussion about such stuff but regarding the C++ linter and the general ambiguity in SC I don't want to open a whole new cans of worms on this.
Just wanted to know if there is an easy way that I maybe did not know yet.